resources/lib/person.py

```diff
@@ -64,11 +66,26 @@ def get_person_details(self):
 
         return li
 
+    def get_combined_list(self, movies, tvshows):
+        combined = movies + tvshows
```

Comment:
`combined = self.result['movies'] + self.result['tvshows']`

Reply:
Yes, but aren't both self.result['movies'] and self.result['tvshows'] lists of xbmcgui objects which aren't easily sortable, plus when I tried using them the the original lists are destroyed breaking control's 10052 and 10053.

Reply:
Should be now problem. I'll merge it and check it on my end.